hw/top_earlgrey/doc/_index.md

```diff
@@ -77,61 +86,11 @@ The block diagram of `top_earlgrey` (the auto-generated module) is shown below.
 In this diagram, the instantiation of the Ibex processor and all of the memories and comportable IPs are shown.
 The IO shown at this level are the internal signals between the IP and the pads instantiated at the target top level netlist.
 
-In the block diagram below, the target netlist `chip_earlgrey_nexsysvideo` is shown, including the FPGA pad names created.
-
-![Top Level Block Diagram](chip_earlgrey_nexsysvideo_block_diagram.svg)
-
-In this diagram, pads for clock and reset are shown, as well as pads for JTAG, SPI device, UART, and GPIO.
-In this platform, at the moment, the logic for the JTAG and the SPI device are multiplexed within `chip_earlgrey_nexsysvideo`.
+In the FPGA platform, the logic for the JTAG and the SPI device are multiplexed within `chip_earlgrey_cw310`.
```

Comment:
would you mind adding a TODO here? this is something we'll need to update once the pin updates are complete.

Reply:
yeah I can rewrite that part.

Reply:
Done. Moved this to a new section as well.